Update the Community page (#996)
kyverno/website: content/en/Community/_index.md

- Monthly on the second Thursday at 7:00 AM PST
- [Repo](https://github.com/nirmata/office-hours-for-kyverno)

### Maintainers Meeting

This is a public weekly meetings focused on topics relevant to plan and discuss upcoming releases, encouraging all contributors to actively participate and contribute to the discussions. It provides an opportunity for maintainers to come together with community members and collaborate on important topics related to the project. By fostering an inclusive environment, the Kyverno team aims to gather diverse perspectives, insights, and ideas that can further enhance the project's development and direction:

- Weekly every Tuesday at 7:30 AM PST
- [Agenda and meeting notes](https://docs.google.com/document/d/1I_GWsz32gLw8sQyuu_Wv0-WQrtRLjn9FuX2KGNkvUY4/edit?usp=sharing)

### Community Meeting

> This is a weekly forum for Kyverno contributors and maintainers to discuss topics relevant to Kyverno development and other contributions. It’s meant for contributors and maintainers to meet, share, and get unblocked, peer review, and discuss all aspects of the Kyverno project:

Reviewer: These descriptions sound too similar and too generic for me to understand. Maintainers Meeting is an internal but "open door" meeting where maintainers discuss issues and PRs pertaining to Kyverno's development and direction. The community are welcome to attend but may not raise agenda items. The Community Meeting is for Kyverno maintainers to make announcements and provide project updates, and this forum allows community members to raise agenda items of any sort, including but not limited to any PRs or issues on which they are working.

Author: I copied it from the meeting doc, how does it look now🤔

- Monthly on the second Thursday at 7:00 AM PST
- [Repo](https://github.com/nirmata/office-hours-for-kyverno)

### Maintainers Meeting

> Maintainers Meeting is an internal but "open door" meeting where  The community are welcome to attend but may not raise agenda items. 

Reviewer: ```suggestion
```

> This is a public weekly meetings for maintainers to discuss issues and PRs pertaining to Kyverno's development and direction. Encouraging all contributors to actively participate and contribute to the discussions but may not raise agenda items. Topics to be proposed by non-maintainers should use the Kyverno Community Meeting as the venue.

Reviewer: ```suggestion
This is a public, weekly meetings for maintainers to discuss issues and PRs pertaining to Kyverno's development and direction. Contributors and end users are welcome to attend and participate but may not raise agenda items. Topics to be proposed by non-maintainers should use the Kyverno Community Meeting.
```

- Weekly every Tuesday at 7:30 AM PST
- [Agenda and meeting notes](https://docs.google.com/document/d/1I_GWsz32gLw8sQyuu_Wv0-WQrtRLjn9FuX2KGNkvUY4/edit?usp=sharing)

### Community Meeting

> This is a public weekly for Kyverno maintainers to make announcements and provide project updates, and this forum allows community members to raise agenda items of any sort, including but not limited to any PRs or issues on which they are working:

Reviewer: ```suggestion
This is a public, weekly for Kyverno maintainers to make announcements and provide project updates, and request input and feedback. This forum allows community members to raise agenda items of any sort, including but not limited to any PRs or issues on which they are working.
```

- Member of the kyverno channel on Kubernetes and/or CNCF Slack
- Attended five (5) Contributors Meetings as documented
- Registered for the Kyverno [mailing list](https://groups.google.com/g/kyverno)

> - Create a pull request to add self to [CODEOWNERS](https://github.com/kyverno/kyverno/blob/main/CODEOWNERS) file in at least one (1) repository

Reviewer: The reason we didn't link to a specific file is because of the clause "in at least one (1) repository." You're now pointing them to a specific one rather than letting them choose a repository. I suppose that's fine, but then the sentence should be updated.

Author: Thanks for pointing it out. Removed links.

**Privileges of a Code Owner**

- Listed as an organization member

> - Listed in [CODEOWNERS](https://github.com/kyverno/kyverno/blob/main/CODEOWNERS) in at least one (1) repository

Reviewer: ditto here and elsewhere
